draft-nottingham-avoiding-internet-centralization.md

```diff
@@ -397,7 +397,7 @@ An important feature of Internet protocols is their ability to evolve over time,
 
 However, protocol extensions can also increase the risk of platform centralization if a powerful entity can change the target for meaningful interoperability by adding proprietary extensions to a standard protocol. This is especially true when the core standard does not itself provide sufficient utility on its own.
 
-For example, the SOAP protocol {{SOAP}} was an extremely flexible framework, allowing vendors to attempt to capture the market by requiring use of their preferred extensions to interoperate.
+For example, extreme flexibility of the SOAP protocol {{SOAP}} allowed vendors to distort the market by requiring use of their preferred extensions to interoperate.
```

Comment:
'distort the market' isn't quite right -- the market hadn't been established yet. Perhaps:

```suggestion
For example, the SOAP protocol's {{SOAP}} extreme flexibility and failure to provide significant standalone value allowed vendors to require use of their preferred extensions, favouring those who had more market power.
```

Reply:
LGTM!